apps/web/screens/Organizations/Organizations.treat.ts

```diff
@@ -0,0 +1,12 @@
+import { style } from 'treat'
+
+export const srOnly = style({
```

Comment:
Maybe this would make sense as a standalone component, or an extension to the Box component, or some other way this could be reusable 😄